azext_iot/product/_help.py

```diff
@@ -0,0 +1,173 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+Help definitions for Product Certification commands.
+"""
+
+from knack.help_files import helps
+
+
+def load_help():
+    helps["iot product"] = """
+        type: group
+        short-summary: Manage device testing for product certification
+    """
+# certification requirements
+    helps["iot product requirement"] = """
+        type: group
+        short-summary: Manage product certification requirements
+    """
+    helps["iot product requirement list"] = """
+        type: command
+        short-summary: Get certification requirements
+        examples:
+        - name: Basic usage
+          text: >
+            az iot product requirement list
+    """
+# product tests
+    helps["iot product test"] = """
+        type: group
+        short-summary: Manage device tests for product certification
+    """
+    helps["iot product test create"] = """
+        type: command
+        short-summary: Create a new product test for product certification
+        examples:
+        - name: Basic usage
+          text: >
+            az iot product test create --configuration-file {configuration_file}
```

Comment:
do the configurations here always come from files or can they be passed inline?

Reply:
My next PR will add to this command to do inline options to build the payload for the service call.